optin/puller/workflows/payout.py

```diff
@@ -110,40 +73,60 @@ async def send_funds(self, destination_address, amount, deadline, nem_address):
 		return transaction_hash
 
 
+async def process_all(processor, unprocessed_requests, sent_requests, deadline, is_dry_run):
+	print(f'processing {len(sent_requests)} SENT requests and {len(unprocessed_requests)} UNPROCESSED requests')
+
+	for address, request_group in unprocessed_requests.items():
+		print()
+
+		processor.is_dry_run = True
+		await processor.process_unprocessed_request(address, request_group, sent_requests, deadline)
+
+		if is_dry_run:
+			continue
+
+		user_input = input('Press Y to send transaction and update databases: ')
```

Comment:
hmm, would maybe just mention, it's ok to Ctrl+C here

Reply:
ctrl+c or any other input (i.e. n) has same effect. ;)